improvement(api-markdown-documenter): Don't support `LineBreakNode`s as `BlockContent` (#24767)
microsoft/FluidFramework: tools/api-markdown-documenter/CHANGELOG.md

# @fluid-tools/api-markdown-documenter

## 0.21.0

### `LineBreakNode` removed from `BlockContent`

> All forms of Block Content are implicitly separated by a line break, so allowing `LineBreakNode`s in that context is redundant.

Author: ```suggestion
Block Content items are implicitly separated by a line break, so allowing `LineBreakNode`s in that context is redundant.
```
